Owncloud iOs Client/Files/Preview/FilePreview/FilePreviewViewController.m

```diff
@@ -1581,9 +1610,10 @@ - (void) errorLogin {
 
 - (void) setNotificationForCommunicationBetweenViews {
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(updatePreviewOverwriteFile:) name:PreviewFileNotification object:nil];
-    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(updatePreviewFileWithNewIDFromDB:) name:uploadOverwriteFileNotification object:nil];
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(updatePreviewFileWithNewIDFromDB:) name:UploadOverwriteFileNotification object:nil];
```

Comment:
I think you should remove the observers UploadOverwriteFileNotification and iPhoneDoneEditFileTextMessageNotification when they're not needed. Probably when the view is not shown